test/test_types_helper.hpp

```diff
@@ -22,6 +22,8 @@
 #include <realm.hpp>
 #include <realm/column_type_traits.hpp>
 
+#include <list>
```

Comment:
It is actually required, there is a use of `std::list` later on. It was working accidentally in the other tests because another include brings in std::list, but the benchmarks don't do this.

Reply:
OK, but it does not show up in the diff, it tricked me.